tools/ea/Ea/Genesis.hs

```diff
@@ -143,8 +144,11 @@ coinContractV4 = "pact/coin-contract/v4/load-coin-contract-v4.yaml"
 coinContractV5 :: FilePath
 coinContractV5 = "pact/coin-contract/v5/load-coin-contract-v5.yaml"
 
-installCoinContractV5 :: FilePath
-installCoinContractV5 = "pact/coin-contract/v5/install-coin-contract-v5.yaml"
+coinContractV6 :: FilePath
+coinContractV6 = "pact/coin-contract/v6/load-coin-contract-v6.yaml"
+
+installCoinContractV6 :: FilePath
+installCoinContractV6 = "pact/coin-contract/v6/install-coin-contract-v6.yaml"
```

Comment:
what's the point of this "install" copy? I actually don't remember this being a thing, but it seems to be redundant in both this case and the previous v5 case.

Reply:
This is for fast-devnet.

Reply:
great, thanks